src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -70,12 +69,11 @@ class FinancialDisclosureSection extends React.Component {
           </div>
 
           <div className="input-section">
-            <ErrorableRadioButtons required
-                errorMessage={validateIfDirty(this.props.data.understandsFinancialDisclosure, isNotBlank) ? '' : 'Please select a response'}
+            <ErrorableCheckbox required
+                errorMessage={isValidFinancialDisclosure(this.props.data) ? '' : 'Please select a response'}
```

Comment:
Does anyone else have a suggestion? I like the above so will change to that unless hear otherwise